src/views/index.html

```diff
@@ -4,194 +4,120 @@
-                                    </div>
-                                </div>
-                            </div>
-                            <div class="two half unit row">
-                                <div class="five unit column">
-                                    <h3>
-                                        Not twice, but three times.<br>Not 2D, but 3D.<br>Not binary, but ternary.
-                                    </h3>
-                                </div>
-                            </div>
-                            <div class="three half unit row">
-                                <div class="five unit column">
-                                    <div id="splash-intro-text">
-                                        <p>
-                                            From the 20<sup>th</sup> to the 21<sup>st</sup> of January 2018, Cambridge University’s annual hackathon yet again brought together 300 hackers, programmers, designers and more from universities all over the world.<br>For 24 hours they built, broke and innovated to produce projects that push the boundaries of technology.
-                                        </p>
-                                        <p>
-                                            Thank you for joining us.
-                                        </p>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                        <div id="intro-social" class="one second-five unit column grows">
-                            <div class="icons persistent row content">
-                                <a class="persistent row centred" rel="external" href="https://www.facebook.com/hackcambridge">
-                                    {{ singleColourIcon(asset, 'Facebook') }}
-                                </a>
-                                <a class="persistent row centred" rel="external" href="https://twitter.com/@hack_cambridge">
-                                    {{ singleColourIcon(asset, 'Twitter') }}
-                                </a>
-                                <a class="persistent row centred" rel="external" href="https://medium.com/hackers-at-cambridge/tagged/hack-cambridge">
-                                    {{ singleColourIcon(asset, 'Medium') }}
-                                </a>
-                                <a class="persistent row centred" rel="external" href="https://www.instagram.com/hack_cambridge/">
-                                    {{ singleColourIcon(asset, 'Instagram') }}
-                                </a>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
+  <main layout="{{ layoutMode }}">
+    <div fixed grid="grid" sidebar columns="7.0" rows="7.0">
+      <img draggable="false" src="{{ asset('images/logo.svg') }}"></img>
```

Comment:
I think the `img` closing tag here might be invalid HTML?

Reply:
Whoops good spot! Maybe I could add a linter to spot these?

Reply:
@jmalcic Think you fixed the other one in your new commit but this one's still incorrect

Reply:
Yeah, I wonder whether there's a way of linting the HTML when it has Nunjucks template tags within it.